openwisp_controller/connection/tests/test_api.py

```diff
@@ -484,6 +503,25 @@ def test_get_deviceconnection_list(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data['count'], 0)
 
+    @patch.object(ListViewPagination, "page_size", 3)
+    def test_deviceconnection_list_ordering(self):
```

Comment:
same here, I would run this as a subtest of `test_get_deviceconnection_list` and simplify the logic to avoid bloat.